ui/src/pages/backoffice/DocumentRequest/DocumentRequestDetails/components/DocumentRequestSteps/components/StepsActions/StepsActions.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Grid, Header, Message } from 'semantic-ui-react';
+import { AcquisitionRoutes, BackOfficeRoutes, ILLRoutes } from '@routes/urls';
+import { documentRequest as documentRequestApi } from '@api';
+import _get from 'lodash/get';
+import { DocumentIcon } from '@pages/backoffice/components';
+import { STEPS } from '../../DocumentRequestSteps';
+import { invenioConfig } from '@config';
+
+const PROVIDERS = invenioConfig.documentRequests.physicalItemProviders;
+
+export default class StepsActions extends Component {
+  render() {
+    const { data, step, fetchDocumentRequestDetails } = this.props;
+    return (
+      <>
+        <DocumentSelection
+          data={data}
+          fetchDocumentRequestDetails={fetchDocumentRequestDetails}
+        />
+        {step !== STEPS.document ? (
+          <ProviderSelection
+            data={data}
+            fetchDocumentRequestDetails={fetchDocumentRequestDetails}
+          />
+        ) : null}
+      </>
+    );
+  }
+}
+
+const ProviderLink = ({ provider }) => {
+  if (provider.pid_type === PROVIDERS.acq.pid_type) {
+    return (
+      <Link to={AcquisitionRoutes.orderDetailsFor(provider.pid)}>
+        {provider.pid}
+      </Link>
+    );
+  }
+  if (provider.pid_type === PROVIDERS.ill.pid_type) {
+    return (
+      <Link to={ILLRoutes.borrowingRequestDetailsFor(provider.pid)}>
+        {provider.pid}
+      </Link>
+    );
+  }
+  return null;
+};
+
+const ProviderHeader = ({ provider }) => {
+  let header = '';
+  if (provider.pid_type === PROVIDERS.acq.pid_type)
+    header = 'Acquisition Order';
+  if (provider.pid_type === PROVIDERS.ill.pid_type)
+    header = 'ILL Borrow Request';
```

Comment:
Just a small comment, IMHO, in general, we should always display `Interlibrary Loan` instead of ILL. I would also use the term `Borrowing` instead of `Borrow` maybe? But my english is crap :)